templates/eth1tx.html

```diff
@@ -166,15 +166,17 @@ <h1 class="h4 mb-1 mb-md-0">
                   {{ if gt (len .InternalTxns) 0 }}
                     <ul class="fa-ul mb-0 mt-2">
                       {{ range $i, $row := .InternalTxns }}
-                        <li class="mb-1">
-                          <i class="fa-li fas fa-long-arrow-alt-right"></i>
-                          <span>Transfer</span>
-                          <span>{{ .Amount }}</span>
-                          <span>From</span>
-                          <span>{{ .From }}</span>
-                          <span>To</span>
-                          <span>{{ .To }}</span>
-                        </li>
+                        {{ if not .Advanced }}
```

Comment:
**issue:** This `Advanced` check was only added for the **Overview** tab here but not for the **Internal Transactions** tab here
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/e553af6244ca1a75c9ee2419afb586d0d94b8942/templates/eth1tx.html#L411